api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
Thanks!

@nikhiljindal In the case of an array, we need to call AllowMultiple on the parameter:
https://github.com/emicklei/go-restful/blob/bdfb7d41639a84ea7c36df648e5865cd9fbf21e2/parameter.go#L81